Source/WTF/wtf/PlatformHave.h

```diff
@@ -958,7 +954,7 @@
 #define HAVE_VISION 1
 #endif
 
-#if (PLATFORM(MAC) && __MAC_OS_X_VERSION_MIN_REQUIRED >= 110000 && __MAC_OS_X_VERSION_MAX_ALLOWED >= 110300) \
+#if PLATFORM(MAC) \
     || (PLATFORM(IOS_FAMILY) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 140500) \
     || (PLATFORM(APPLETV) && __TV_OS_VERSION_MAX_ALLOWED >= 140500) \
     || (PLATFORM(WATCHOS) && __WATCH_OS_VERSION_MAX_ALLOWED >= 70400)
```

Comment:
:(

Reply:
Is the frown because I only touched Mac and not the other platforms?

Reply:
Yes.

Reply:
Okay, will address this in a follow-up.